src/main/scala/chisel3/experimental/util/algorithm/Bitwise.scala

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.experimental.util.algorithm
```

Comment:
is there a reason this is `experimental`? And what is defining characteristic of what makes a `util` an `algorithm`?

Reply:
I'm confident enough that this is a useful and correct API to argue it doesn't need `experimental`. I also think `algorithm` is unnecessary but don't feel too strongly either way. Directly in `chisel3.util` might improve discoverability although we probably ought to just put some of these into a doc on the webpage.

Reply:
+1 on `util`